src/jinja2_fragments/__init__.py

```diff
@@ -85,3 +85,57 @@ def render_block(
         return environment.concat(block_render_func(ctx))  # type: ignore
     except Exception:
         environment.handle_exception()
+
+
+def render_block_list(
+    environment: Environment,
+    template_name: str,
+    block_name_list: list[str],
+    *args: typing.Any,
+    **kwargs: typing.Any,
+) -> str:
+    """This returns one or more rendered template blocks as a string."""
+    if environment.is_async:
+        import asyncio
+
+        close = False
+
+        try:
+            loop = asyncio.get_running_loop()
+        except RuntimeError:
+            loop = asyncio.new_event_loop()
+            close = True
```

Comment:
As we have this snippet on `render_block`, it makes me think if we should move it to a `_get_loop` function... thoughts?

Reply:
It does make sense to me